### ISSUE TITLE
Invalid output when a parameter description contains a "."

### ISSUE THREAD

**Issue by axelhzf** [drwpow/openapi-typescript#985]
**Description**

When a parameter description contains a "." at the end, it produces an invalid typescript output. Here is a small schema that produce an invalid output

```yml
openapi: 3.0.0
info:
  title: A
  version: '1.0'
paths:
  '/programs/{id}/data':
    parameters:
      - name: id
        in: path
        schema:
          type: string
        required: true
        description: Description.
    get:
      responses:
        200:
          description: OK.
          content:
            "application/json":
              schema:
                type: string
```

The code generated is

```ts
export interface paths {
  "/programs/{id}/data": {
    get: {
      responses: {
        /** @description OK. */
        200: {
          content: {
            "application/json": string;
          };
        };
      };
    };
    {
    parameters: {
        /** @description Description. */
      path: {
        id: string;
      };
    };
  }
  };
}
```

There are an extra `{}` around `parameters`. If I remove the "." at the end of the description, the generation works correctly.

| Name                 | Version                      |
|:---------------------|:-----------------------------|
| `openapi-typescript` | `6.0.2`                      |
| Node.js              | `16.17.1`                      |
| OS + version         | `macOS 13` |

**Reproduction**

Generate the types from the previous schema

**Expected result**

_(in case it’s not obvious)_

**Checklist**

- [x] I’m willing to open a PR (see [CONTRIBUTING.md](https://github.com/drwpow/openapi-typescript/blob/main/CONTRIBUTING.md))

**Comment by axelhzf** [comment]
I think the problem is related to this unwrap_obj_re here

https://github.com/drwpow/openapi-typescript/blob/main/src/transform/path-item-object.ts#L48
https://github.com/drwpow/openapi-typescript/blob/main/src/transform/path-item-object.ts#L12

**Comment by axelhzf** [comment]
Fixed by #986
